spec/models/com_effort_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe ComEffort, type: :model do
+  describe 'columns' do
+    it { is_expected.to have_db_column(:id).of_type(:integer).with_options(null: false) }
+    it { is_expected.to have_db_column(:com_id).of_type(:string) }
+    it { is_expected.to have_db_column(:course_year).of_type(:string) }
+    it { is_expected.to have_db_column(:course).of_type(:string) }
+    it { is_expected.to have_db_column(:event_type).of_type(:string) }
+    it { is_expected.to have_db_column(:faculty_name).of_type(:string) }
+    it { is_expected.to have_db_column(:event).of_type(:string) }
+    it { is_expected.to have_db_column(:event).of_type(:string) }
+    it { is_expected.to have_db_column(:hours).of_type(:integer) }
+
+    #it { is_expected.to have_db_index([:academic_course_id, :term, :calendar_year]).unique(true) }
+  end
+
+  # describe 'associations' do
+  #   it { is_expected.to have_many(:faculties) }
+  #   it { is_expected.to have_many(:sections) }
+  # end
```

Comment:
We want to get the index test working here, add the faculty_id column test, and get the association tests working.

Reply:
[4664f7b](https://github.com/psu-libraries/fams_tools/pull/46/commits/4664f7ba4375124c0d07df8d4907e5a4e952a2a1) and [4d1d847](https://github.com/psu-libraries/fams_tools/pull/46/commits/4d1d847bb53b04693bc737c9586da111e910da51)